package.json

```diff
@@ -46,7 +46,7 @@
     "to-markdown": "3.1.0",
     "ts-loader": "5.3.3",
     "ts-node": "7.0.1",
-    "typescript": "3.5.1",
+    "typescript": "3.5.0",
```

Comment:
Yikes, Looks like I make this commit wrong. I put this version and ran `yarn install` and it prompted me that this version is not valid so I picked v3.4.5. I though yarn would actually update this but it didn't and I lost track this. (And provision afterwards I think ran fine for me since it was cached)